app/models/devise_token_auth/concerns/user_omniauth_callbacks.rb

```diff
@@ -5,7 +5,7 @@ module DeviseTokenAuth::Concerns::UserOmniauthCallbacks
 
   included do
     validates :email, presence: true,if: :email_provider?
-    validates :email, 'devise_token_auth/email' => true, allow_nil: true, allow_blank: true, if: :email_provider?
+    validates :email, :devise_token_auth_email => true, allow_nil: true, allow_blank: true, if: :email_provider?
```

Comment:
What does it fixes specifically? Because this was introduced for conflicts with other lib: #1233

Reply:
@skycocker will this conflict with solidus/spree ?

Reply:
yes, this will be a problem with pretty much any other lib which has a validator called `EmailValidator`. Maybe I introduced another bug I wasn't aware of? But it's hard to tell from just this revert.

Reply:
Conflict EmailValidator in my rails app.

Reply:
I didn't receive a response so I'm going to merge this